src/lib/EventListener/SetViewParametersListener.php

```diff
@@ -276,7 +277,18 @@ private function getIgnoredContentFields(array $fieldsData): array
             $ignoredFieldIdentifiers[] = $fieldIdentifier;
         }
 
-        return $ignoredFieldIdentifiers;
+        $metaFieldGroups = $this->configResolver->getParameter(
+            'admin_ui_forms.content_edit.meta_fieldgroup_list'
+        );
+        $metaFieldIdentifiers = array_keys(
+            array_filter(
+                $fieldsData,
+                static fn (FormInterface $field): bool => true
+                    === in_array($field->getData()->fieldDefinition->fieldGroup, $metaFieldGroups)
```

Comment:
Nitpicking here but this seems to be more readable to me:
```suggestion
                static fn (FormInterface $field): bool => 
                    true === in_array($field->getData()->fieldDefinition->fieldGroup, $metaFieldGroups)
```

Reply:
@webhdx had the same idea but unfortunately, CS Fixer places it in the same line.

Reply:
Actually made it work differently :)